### PR TITLE
Basemaps helper for CARTO module

### PR DESCRIPTION
**Pull request by alasarr**
This include a module with an object to avoid typing the long URL of the basemaps

### CODE REVIEW

File: modules/carto/src/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export {getDefaultCredentials, setDefaultCredentials} from './auth.js';
 export {default as CartoSQLLayer} from './layers/carto-sql-layer';
 export {default as CartoBQTilerLayer} from './layers/carto-bqtiler-layer';
+export {default as basemaps} from './basemaps';
```
Comment (Pessimistress) [inline modules/carto/src/index.js:4]: deck.gl uses all CAPS for constant names.

Comment (alasarr) [reply modules/carto/src/index.js]: Done! Thanks for the tip!

**Comment by alasarr** [comment]
I've added also a doc section

File: docs/api-reference/carto/basemap.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    controller: true
+  });
+```
+
+## Supported basemaps
+
+There are three basemaps available today:
+
+* POSITRON
```
Comment (ibgreen) [inline docs/api-reference/carto/basemap.md:52]: A table with small screenshots would be really nice.

**Review by ibgreen — APPROVED** [review]
Very nice!

On a side note I wonder what the experience of using these basemaps with our TileLayer would be like? What is the feature delta (labels, multi layer support etc) 

Just thinking it would be cool to be able to have a way to use basic free basemaps like these with pure deck.gl (not even using react-map-gl).

File: docs/api-reference/carto/basemap.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Basemap
```
Comment (Pessimistress) [inline docs/api-reference/carto/basemap.md:1]:
```suggestion
# BASEMAP
```

File: docs/api-reference/carto/basemap.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+To install the dependencies from NPM:
+
+```bash
+npm install deck.gl
+# or
+npm install @deck.gl/core @deck.gl/layers @deck.gl/carto
+```
+
+```js
+import {BASEMAP} from '@deck.gl/carto';
```
Comment (Pessimistress) [inline docs/api-reference/carto/basemap.md:16]:
```suggestion
import {DeckGL} from '@deck.gl/react';
import {StaticMap} from 'react-map-gl';
import {BASEMAP} from '@deck.gl/carto';
```

**Review by Pessimistress — APPROVED** [review]

File: docs/api-reference/carto/basemap.md (hunk 2 above)
Comment (alasarr) [reply docs/api-reference/carto/basemap.md:54]: Done!

**Pull request merged**